### PR TITLE
Map on Event Response Screen

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
Added a button that pulls up a map in a modal

screenshot 2018-06-09 14 29 57
screenshot 2018-06-09 14 29 45

### CODE REVIEW

File: client/src/components/Modal/MapModal.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+import styles from './styles';
+
+const screen = Dimensions.get('window');
+
+const ASPECT_RATIO = screen.width / screen.height;
+const LATITUDE_DELTA = 0.0922;
+const LONGITUDE_DELTA = LATITUDE_DELTA * ASPECT_RATIO;
+
+const mapModal = ({
```
Comment (sdunster) [inline client/src/components/Modal/MapModal.js]: `MapModal` please - this name does actually show up in the component hierarchy I'm pretty sure so it's important to capitalise it.

**Comment by sdunster** [comment]
This diff is still titled WIP.

**Comment by sdunster** [comment]
Please dont forget to squash into a commit with a good commit message/description.

File: client/src/components/Modal/MapModal.js (hunk 1 above)
Comment (OSPFNeighbour) [reply client/src/components/Modal/MapModal.js]: Nice catch, thanks

Comment (OSPFNeighbour) [reply client/src/components/Modal/MapModal.js]: Nice catch, thanks

**Comment by OSPFNeighbour** [comment]
Still sorta working on it so it's wip

File: client/src/components/Modal/MapModal.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  markers,
+}) => (
+  <View>
+    <Modal
+      transparent
+      visible={visible}
+      onRequestClose={closeModal}
+      animationType="fade"
+    >
+      <TouchableWithoutFeedback onPress={() => closeModal}>
```
Comment (OSPFNeighbour) [inline client/src/components/Modal/MapModal.js]: @sdunster how did you let that slip past!

we should have a linter rule to catch these

Comment (sdunster) [reply client/src/components/Modal/MapModal.js]: ooh yeah ouch. hard to lint on this stuff because that might be entirely sensible way of doing things

**Comment by sdunster** [comment]
rebase

File: client/src/components/Modal/MapModal.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+              </View>
+              <View style={styles.mapContainer}>
+                <View>
+                  <View>
+                    {/* TouchableWithoutFeedback prevents touch propagation */}
+                    <TouchableWithoutFeedback>
+                      <MapView
+                        onMapReady={this.zoomMap}
+                        ref={(ref) => {
+                        this.map = ref;
```
Comment (sdunster) [inline client/src/components/Modal/MapModal.js]: here until L77 the indentation is fucked up please fix

Comment (OSPFNeighbour) [reply client/src/components/Modal/MapModal.js]: got it

**Pull request merged**